jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerCredentialHelper.java

```diff
@@ -70,12 +70,11 @@
    *
    * @return the Docker credentials by calling the corresponding CLI
    * @throws IOException if writing/reading process input/output fails.
-   * @throws NonexistentServerUrlDockerCredentialHelperException if credentials are not found.
+   * @throws UnknownServerUrlException if credentials are not found.
```

Comment:
This exception description doesn't seem quite right.  _if no credentials could be found for the corresponding server_. Hmm, I wonder if _Unknown_ is right… Does _Unhandled_ seem more apt?

Reply:
Hmm yeah, I think that does make more sense.